modules/contacts/client/components/RemoveContact.js

```diff
@@ -32,10 +40,10 @@ export function RemoveContact({ t, contact, show, inProgress, onHide, onRemoveCo
   const labels = situationLabels[situation];
 
   return (
-    <Modal show={show} onHide={onHide}>
+    <Modal show={show} onHide={onCancel}>
       <div className="modal-content">
         <Modal.Header>
-          <button type="button" className="close" aria-hidden="true" onClick={onHide} ng-if="!removeContactModal.isLoading">&times;</button>
+          <button type="button" className="close" aria-hidden="true" onClick={onCancel} ng-if="!removeContactModal.isLoading">&times;</button>
```

Comment:
Remember to remove `ng-if` Angular bits. :-)

Reply:
Thank you @simison! :+1:
Fixed in a98e3403d544775fdae6d8e06074382cdabc9a46 as part of #1121.